kubernetes/api/v1alpha1/servicelevelobjective_types.go

```diff
@@ -89,13 +89,13 @@ type ServiceLevelIndicator struct {
 	Ratio *RatioIndicator `json:"ratio,omitempty"`
 
 	// +optional
-	// Latency is the indicator that measures a certain percentage to be fast than.
+	// Latency is the indicator that measures a certain percentage to be faster than.
 	Latency *LatencyIndicator `json:"latency,omitempty"`
 
 	// +optional
-	// BoolGauge is the indicator that measures wheter a boolean gauge is
-	// successul.
-	BoolGauge *BoolGaugeIndicator `json:"bool_gauge,omitempty"`
```

Comment:
Do I understand correctly that removing `BoolGauge` was a mistake? 😅

Reply:
Good catch. I need to clean that up. It was a rebase mistake indeed.

Reply:
Done. It's only additions to these files now and the BoolGauge is left untouched. :slightly_smiling_face: